test/integration/nighthawk_grpc_service.py

```diff
@@ -39,6 +39,7 @@ def __init__(self,
     assert ip_version != IpVersion.UNKNOWN
     self.server_port = 0
     self.server_ip = server_ip
+    self.log_lines = None
```

Comment:
(nit) Does this need to be public or would self._log_lines do? If public, can we document it above in the Attributes: section of the class docstring?

Reply:
Well, it gets consumed over at https://github.com/envoyproxy/nighthawk/pull/363/files#diff-9620177d1b84e578a841abdc925dcdf4R33. So it needs to be public, and documented. I'll add that.

Reply:
Done in 593fe3c (also moved the line that resets the log to `None` to a better place).